src/sections/Meshery/Meshery-integrations/Individual-Integrations/howItWork.js

```diff
@@ -7,7 +7,7 @@ import SMPImage from "../../images/meshery_multi_mesh.png";
 
 const HowIntegrationWorksWrapper = styled.section`
 
-  background: #F7F7F7;
+  background: ${props => props.theme.DarkTheme};
```

Comment:
You need to switch the color based on this value. Not, just return the value.

Reply:
Can you please elaborate. The background color is getting changed based on the theme selection.

Reply:
Well, the section color is not changing but the background color of the complete HTML is and the same is reflecting in the section.
Update the code to `background-color: ${props => props.theme.DarkTheme ? #101010 : #F7F7F7};`

Reply:
Also, while you are making the change can you please apply a similar update to the link color of `See Documentation` text?
Use `colorr: ${props => props.theme.text };` It should do.

Reply:
Thanks for the detailed explanation. I got it now. Can you please review new changes.